Feature: Icons added for new nationalrailtimes custom component integ…

crismc opened the pull request:
…ration

<!--
  You are amazing! Thanks for contributing to our project!
  Please, DO NOT DELETE ANY TEXT from this template! (unless instructed).
-->
## Proposed change
<!-- 
  This adds icons for a new custom integration [nationalrailtimes](https://github.com/crismc/homeassistant_nationalrailtimes_integration).
  This integration provides departure, journey and arrival times from selected UK based National Rail stations
-->


## Type of change
<!--
  What type of change does your PR introduce to the Home Assistant Brands?
  NOTE: Please, check only 1! box! 
  If your PR requires multiple boxes to be checked, you'll most likely need to
  split it into multiple PRs. This makes things easier and faster to code review.
-->

- [ ] Add a new logo or icon for a new core integration
- [ ] Add a missing icon or logo for an existing core integration
- [x] Add a new logo or icon for a custom integration (custom component)
  - [ ] I've opened up a PR for my custom integration on the [Home Assistant
    Python wheels repository](https://github.com/home-assistant/wheels-custom-integrations)
- [ ] Replace an existing icon or logo with a higher quality version
- [ ] Removing an icon or logo

## Additional information
<!--
  Details are important, and help maintainers processing your PR.
  Please be sure to fill out additional details, if applicable.
-->

- This PR fixes or closes issue: fixes #
- Link to code base pull request: 
- Link to documentation pull request: 
- Link to integration documentation on our website: 

## Checklist
<!--
  Put an `x` in the boxes that apply. You can also fill these out after
  creating the PR. If you're unsure about any of them, don't hesitate to ask.
  We're here to help! This is simply a reminder of what we are going to look
  for before merging your contribution.
-->

- [x] The added/replaced image(s) are **PNG**
- [x] Icon image size is 256x256px (`icon.png`)
- [x] hDPI icon image size is 512x512px for  (`icon@2x.png`)
- [x] Logo image size has min 128px, but max 256px, on the shortest side (`logo.png`)
- [x] hDPI logo image size has min 256px, but max 512px, on the shortest side (`logo@2x.png`)

<!--
  Thank you for contributing <3
-->

frenck reviewed (changes requested):
These images do not seem to match the branding used by the data source:

<img width="563" alt="CleanShot 2022-11-15 at 20 33 29@2x" src="https://user-images.githubusercontent.com/195327/202009369-f0ee3dbb-516b-4d84-9e2c-f01f9f18f316.png">

crismc commented:
National rails brand is the arrows. This is traditionally recognised as the red arrows, although the website isn't focused on the colour:
<img width="100" alt="image" src="https://user-images.githubusercontent.com/1785584/202010001-bedb563c-6d61-4797-acbc-2bf7229a4832.png">

The logos submitted incorporate this primary brand.

Unless the requirement is to only be the data source?

frenck commented:
It fine, as long as they are used by the provider. Could you point me to an official resource that uses these?

crismc commented:
[www.nationalrail.com](https://www.nationalrail.com) has the core logo leveraged for this submission.
The wrapper (i.e. the encapsulating train icon) is reference to my component leveraging this data source.

frenck commented:
Thanks, that website uses a different brand asset:

<img width="402" alt="image" src="https://user-images.githubusercontent.com/195327/202011760-fc71cf34-cf78-42f0-b7dd-a72d2c7e8f10.png">

Which is also used on their social media:
<img width="359" alt="CleanShot 2022-11-15 at 20 46 45@2x" src="https://user-images.githubusercontent.com/195327/202011787-2246412c-4e6a-4c83-bf73-98a3eabcfa8e.png">

<img width="715" alt="CleanShot 2022-11-15 at 20 47 30@2x" src="https://user-images.githubusercontent.com/195327/202011822-cbcbdf24-ac2b-41fe-b94b-0cc3fbd2a8a4.png">

I do not see the exact icons/images provided here in this PR. Could you provide me an exact link that displays these brand assets?

crismc commented:
Perhaps the brand guidelines may be better to reference the original logo?
[www.nationalrailguidelines.co.uk PDF]( http://www.nationalrailguidelines.co.uk/download.php?filename=pdfs/NR012.pdf)

Happy to change to only the respective icon indicated in these guidelines if the encapsulating train is not acceptable?

frenck commented:
Thanks, but non of the images provided in this PR is in that document either.

../Frenck

crismc commented:
I believe you're asking for only the brand logo, without the train encapsulation.
In the interest of everyones sanity, I will update the icons accordingly to only the NationalRail icon :)

frenck commented:
> I believe you're asking for only the brand logo, without the train encapsulation.

If that is what these logos are... maybe yes? I don't know. I'm just trying to verify that the provided images are actual brand images used by the data provider or manufacturer of the devices.

I could not find these that are in this PR, thus me (repeatedly asking) for a reference to an official resource where these images are used. As of yet, you haven't been able to provide that.

../Frenck

crismc commented:
I have updated the logos to the data source.

For clarity on the previous submission, I assumed there was a hint of creativity allowed with the component logos, hense the encapsulation.
The 2 arrows within the train were the recognised National Rail logos present in the respective brand guidelines, and the encapsulating train was my own "salt" on top.

<img width="400" alt="image" src="https://user-images.githubusercontent.com/1785584/202022364-401a6be7-d9d4-4100-b481-4b644158172e.png">

Either way, moot point, and updated to be only the respective data source brand. Apologies if any frustration was caused. :)

frenck reviewed (approved):
No problem!

Thanks, @crismc 👍 

../Frenck
